.github/workflows/build.yml

```diff
@@ -80,3 +80,8 @@ jobs:
         DB_PORT: 15432
         DB_USER: postgres
         DB_PASSWORD: postgres
+    - name: Upload coverage to codecov
+      uses: codecov/codecov-action@v3
+      with:
+        fail_ci_if_error: true
+        verbose: true
```

Comment:
```suggestion
        verbose: true
        flags: unittests
```

Reply:
@kgaikwad Although this is an optional suggestion, IMO it helps provide a legible context/metadata for the coverage data